src/algorithm/orient.rs

```diff
@@ -29,6 +29,20 @@ pub trait Orient<T> {
     /// assert_eq!(oriented.interiors[0].0, oriented_int_ls.0);
     /// ```
     fn orient(&self, orientation: Direction) -> Self;
+
+    /// Orient a Polygon according to default orientation
+    fn orient_default(&self) -> Self
```

Comment:
I personally think it's a little redundant having these new methods in addition to `orient`, but if people find it helpful then I'm cool with it

Reply:
Yeah, I think it should be one or the other – I'm not tied to using an enum, though.